src/components/dataset/schemaCreation/DatasetSchemaCreationView.tsx

```diff
@@ -75,93 +90,139 @@ interface TabConfig {
-          tables and columns within the tables - and any associations between columns in separate
-          tables, if multiple tables contain the same data category (for instance, if you have a
-          "subject" table and a "sample" table, and both tables contain a column of the same subject
-          IDs).
-          <Tabs classes={{ root: classes.tabsRoot }} value={value} onChange={handleChange}>
+      <FormProvider {...formMethods}>
+        <form className={classes.contentContainer} onSubmit={handleSubmit(onSubmit)}>
+          <div className={classes.mainContent}>
+            <Typography variant="h3" className={classes.pageTitle}>
+              Create a dataset schema for ingesting data
+            </Typography>
+            Before you can ingest data files, you need to define the structure of the data you'll be
+            ingesting by specifying the schema of the data. The schema is a template for the data
+            you'll ingest later. You'll specify the number and names of the data categories - the
+            tables and columns within the tables - and any associations between columns in separate
+            tables, if multiple tables contain the same data category (for instance, if you have a
+            "subject" table and a "sample" table, and both tables contain a column of the same
+            subject IDs).
+            <Tabs classes={{ root: classes.tabsRoot }} value={currentTab} onChange={changeTab}>
+              {tabs.map((tabConfig: TabConfig, i: number) => (
+                <Tab
+                  key={`dataset-schema-creation-tab-${i}`}
+                  label={
+                    <div>
+                      <Typography variant="h3">Step {i + 1}</Typography>
+                      <div className={classes.tabDescription}>{tabConfig.description}</div>
+                    </div>
+                  }
+                  className={classes.tabRoot}
+                  disableFocusRipple
+                  disableRipple
+                />
+              ))}
+            </Tabs>
             {tabs.map((tabConfig: TabConfig, i: number) => (
-              <Tab
-                key={`dataset-schema-creation-tab-${i}`}
-                label={
-                  <div>
-                    <Typography variant="h3">Step {i + 1}</Typography>
-                    <div className={classes.tabDescription}>{tabConfig.description}</div>
-                  </div>
-                }
-                className={classes.tabRoot}
-                disableFocusRipple
-                disableRipple
-              />
-            ))}
-          </Tabs>
-          {tabs.map((tabConfig: TabConfig, i: number) => (
-            <TabPanel value={value} index={i} key={`dataset-schema-creation-tabpanel-${i}`}>
-              {tabConfig.content}
-            </TabPanel>
-          ))}
-        </div>
+              <div key={`dataset-schema-creation-tabpanel-${i}`} hidden={currentTab !== i}>
```

Comment:
I would recommend pulling this out into a component. There's enough display complexity here that it might help with understandability to pull it out. I also think that this is very reusable if you have other tab based forms.

Reply:
Would you mind clarifying which part should be a new component? I have it set up rn so the tab contents are separate components... do you mean the tabs should be its own one too?

Reply:
I think that the ternary content seems fairly standard (with the stepping through tabs). Because of that, I would pull the div out that starts on line 161.

Reply:
Turns out it already exists as a component! 🎉